codestodians/food-basket-front-end
# [Ft: 151044794] Graphql frontend integration

#### What does this PR do?
- This PR adds graphql to the frontend.

#### Description of Task to be completed?
- Update application entry point to include the apollo client and the apollo provider.
- Add windows dev tools.
- Add Navbar.
- Add Landing Page and 404 component.
- Add action to set token in local storage.
- Add signin and sign up form components.
- Add signup mutation to post user to the api.
- Add relevant styles.

#### How should this be manually tested?
- Run `yarn` to install additional dependencies.
- Run `yarn start` to start your server, you should see the screen shown in the screenshot section.
- Fill the sign up form and click submit. The user data should be added to your database, user is redirected to the landing page and token is set in local storage.

#### Any background context you want to provide?
- Page designs are in the interim, would be removed once the mock is available.
- If running locally, ensure your graphql server is running.

#### What are the relevant pivotal tracker stories?
[#151044794](https://www.pivotaltracker.com/story/show/151044794)

#### Screenshots (if appropriate)

- Landing Page

![screen shot 2017-09-21 at 11 03 23 am](https://user-images.githubusercontent.com/20999363/30690578-92f6056e-9ebc-11e7-91ae-94d06dae040e.png)

- Home Page

![screen shot 2017-09-21 at 11 11 07 am](https://user-images.githubusercontent.com/20999363/30690895-a714f982-9ebd-11e7-80ed-0e1b7007f35d.png)
